metapkg/packages/rust.py

```diff
@@ -41,9 +42,17 @@ def get_build_install_script(self, build) -> str:
             target = "--target x86_64-unknown-linux-musl"
         else:
             target = ""
+        if sys.platform == "darwin":
+            sed = "gsed"
+            install_gsed = "brew install gnu-sed"
+        else:
+            sed = "sed"
+            install_gsed = ""
+
         return textwrap.dedent(
             f"""\
-            sed -i -e '/\\[package\\]/,/\\[.*\\]/{{
+            {install_gsed}
```

Comment:
This should be done in `MacOSTarget.prepare` instead.

Reply:
Yeah that was my initial attempt, but looks like the CLI is building with `--generic`. Removing `--generic` generated a macOS pkg for me

Reply:
`GenericTarget.prepare()` then